### PR TITLE
Show numerical values when the user clicks on the map

### PR DESCRIPTION
**Pull request by julienrf**
Demo:

demo-popup

- Now, when the users click on the map, they see a popup with numerical values of what is displayed on the map overlay
- When they click again, they see the meteogram/sounding
- The color of the “XC flying potential index” is not interpolated anymore to mimic the behavior of the current soaringmeteo (see commit a925bc3fa2395f37957b9b6db450ba804dacb13c). I am still not sure whether this is the best idea or not… If we do it for that index, maybe we should do it for all the other variables (boundary layer depth, thermals velocity, etc.?)

Please let me know what you think about it and I will update the documentation accordingly.

### CODE REVIEW

**Comment by Boran** [comment]
It's a good idea indeed .
How about adding the datetime to the popup, to create a logical link to the date selection you show in the demo video?

**Comment by julienrf** [comment]
Thank you for the suggestion Sean, what about the following?

other-demo

**Comment by Boran** [comment]
Very nice!

File: frontend/src/layers/ThermalVelocity.tsx [hunk 1]
```diff
@@ -11,6 +13,15 @@ export class ThermalVelocity {
     ctx.fillStyle = `rgba(${color.red}, ${color.green}, ${color.blue}, 0.25)`;
     ctx.fillRect(topLeft.x, topLeft.y, bottomRight.x - topLeft.x, bottomRight.y - topLeft.y);
   }
+
+  summary(forecastPoint: ForecastPoint): JSX.Element {
+    return <table>
+      <tbody>
+        <tr><th>Thermals velocity: </th><td>{ forecastPoint.thermalVelocity }&nbsp;m/s</td></tr>
```
Comment (Boran) [inline frontend/src/layers/ThermalVelocity.tsx:20]: no "s" on "Thermals velocity" ?

Comment (julienrf) [reply frontend/src/layers/ThermalVelocity.tsx]: Thank you, fixed in [d0d7bad](https://github.com/soaringmeteo/soaringmeteo/pull/39/commits/d0d7bada6bb4414f3e80df3e357b9bc8af272f69)

**Pull request merged**